shell/scripts/test-plugins-build.sh

```diff
@@ -141,6 +141,8 @@ if [ "${SKIP_STANDALONE}" == "false" ]; then
   pushd test-app > /dev/null
 
   yarn install
+  # this is the "same" as doing a yarn dev (in a build sense)
+  # it's to make sure the dev environment is running properly
```

Comment:
Is this comment meant for a different PR?

Reply:
This was discovered during the work on this issue because of a possible bug and I just wanted to add a comment to something that we didn't understand the purpose, for future. I don't see any harm on this going in so I'll keep it.